<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="childrenCollapsible">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="controlComboBox"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Style:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="styleComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QQuickWidget" name="quickWidget">
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
       <property name="source">
        <url>
         <string/>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
